src/ontology/uberon-edit.obo

```diff
@@ -53,6 +54,7 @@ treat-xrefs-as-equivalent: OG
 treat-xrefs-as-equivalent: VSAO
 treat-xrefs-as-is_a: VHOG
 import: http://purl.obolibrary.org/obo/uberon/components/mappings.owl
+import: http://purl.obolibrary.org/obo/uberon/components/vasculature_class.owl
```

Comment:
@anitacaron if you add this, the owl file has to exist as well! So yes, leave it checked in here as is!